src/bridge.ts

```diff
@@ -41,10 +41,28 @@ const getChannel = <A, B>(channel: IPC_CHANNELS) => {
   // @todo: CHECK CHANNEL VALIDITY
 
   const asRenderer = (ipcRenderer: IpcRenderer) => ({
+    emit: (...args: unknown[]) => ipcRenderer.send(channel, ...args),
+    on: (listener: (event: IpcRendererEvent, ...args: unknown[]) => void) => {
+      ipcRenderer.on(channel, listener);
+
+      return () => {
+        ipcRenderer.removeListener(channel, listener);
+      };
+    },
     invoke: (request: A) => ipcRenderer.invoke(channel, request)
   });
 
-  const asMain = (ipcMain: IpcMain) => ({
+  const asMain = (ipcMain: IpcMain, Contents: typeof WebContents) => ({
+    emit: (...args: unknown[]) => {
+      Contents.getAllWebContents().forEach((webContents) => webContents.send(channel, ...args));
```

Comment:
I would prefer using the one exposed by main only. This could be a security issue.